docs/data/date-pickers/shortcuts/OnChangeShortcutLabel.tsx

```diff
@@ -63,6 +87,14 @@ const shortcutsItems: PickersShortcutsItem<Dayjs | null>[] = [
       return getMonthWeekday(8, 1, 1);
     },
   },
+  {
+    label: 'Halloween',
```

Comment:
WDYT about adding more items into the [Customization](https://deploy-preview-13972--material-ui-x.netlify.app/x/react-date-pickers/shortcuts/#customization) example with horizontal shortcuts and adjusting the demo such that it would have a horizontal scrollbar. 🤔 
This would also be a step in the right direction to fix this page behavior on mobile. 🙈

Reply:
Done in [f5577a1](https://github.com/mui/mui-x/pull/13972/commits/f5577a1b3841337b85940b23c08e5454e352ab2a) 
About the behavior on mobile, do you suggest to conditionally manipulate the number of calendars or not using the static picker for mobile at all ?

Reply:
> About the behavior on mobile, do you suggest to conditionally manipulate the number of calendars or not using the static picker for mobile at all ?

WDYT about adding an overflow(X)? 🤔 
That's the easiest we could do without too much magic. 👍